### DISCUSSION TITLE
Status Check - Self Signed Certificate in Chain (with local CA)

### DISCUSSION THREAD

**Discussion by nicedevil007** [Lissy93/dashy#1422] [Help and Support]
Hey guys,

I installed Dashy on my AlmaLinux 9.3 LXC container and it is doing a great job so far. Now I struggle with the health/status check.
F.e. I have https://npm.mylocaldomain.local (NginxProxyManager) with a self signed certificate in front of it, but it is not selfsigned by the LXC of NginxProxyManager it is signed by my own local CA.

The local CA's root Certificate is imported to the root store but the error message keeps the same.

<img width="229" alt="image" src="https://github.com/Lissy93/dashy/assets/17103076/b8af0072-f363-4c1f-90ba-80d84c650be0">

And yes I tried rebooting the service and the whole LXC container, doesn't make any difference.
I hope someone else was able to do these checks with a local CA (skipping the SSL Cert isn't a solution). Maybe Dashy is using any other kind of Root Store for the CAs?

**Comment by rmdes** [comment]
I have the same issue and wondering if by adding the self-signed certificate of the target URL's into the root store CA of the container wouldn't be the fix for this...
